app/cli/cmd/attestation_add.go

```diff
@@ -47,6 +47,16 @@ func newAttestationAddCmd() *cobra.Command {
 		Annotations: map[string]string{
 			useWorkflowRobotAccount: "true",
 		},
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			switch {
+			case name == "" && kind == "":
+				return fmt.Errorf("both --name and --kind cannot be empty")
```

Comment:
this doesn't sound right, but it might be. fwiw, I'd have wrote it as 

either --name or --kind needs to be set